doc/policy/mempool-replacements.md

```diff
@@ -53,12 +46,10 @@ other consensus and policy rules, each of the following conditions are met:
    significant portions of the node's mempool using replacements with multiple directly conflicting
    transactions, each with large descendant sets.
 
-6. The replacement transaction's feerate is greater than the feerates of all directly conflicting
+5. The replacement transaction's mining score is greater than the mining score of all directly conflicting
```

Comment:
bf467f8286425b692a1736ab6d417d0ba6074658 Maybe make this rule 7. That seems a bit more clear than saying "previously this rule referred to fee rate"